zio-http/shared/src/main/scala/zio/http/Routes.scala

```diff
@@ -282,6 +282,10 @@ object Routes extends RoutesCompanionVersionSpecific {
   def singleton[Env, Err](h: Handler[Env, Err, (Path, Request), Response])(implicit trace: Trace): Routes[Env, Err] =
     Routes(Route.route(RoutePattern.any)(h))
 
+  implicit class RouteOps[-Env, +Err <: Response](val routes: Routes[Env, Err]) extends AnyVal {
```

Comment:
By the way sorry if this a stupid question, but do we need the implicit class? Isn't possible to have this as a method directly implemented on Routes instead or am I missing something?

Reply:
Just wanted to have implementations of server far from route implementation, mb I'm wrong

Reply:
This seems to be valid, but I am uncertain if it provides any significant advantages.

Reply:
Should I move it into implementation?

Reply:
No implicit. Just a method on routes

Reply:
I've done it, but now it's failing with handshake test for some reason